examples/watch_imu.c

```diff
@@ -203,37 +188,35 @@ int main(int argc, const char* argv[])
     // Register some callbacks.
     mip_dispatch_handler packet_handler;
     mip_dispatch_handler data_handlers[4];
-    mip_interface_register_packet_callback(&device, &packet_handler, MIP_DISPATCH_ANY_DATA_SET, false, &handlePacket,
-                                           NULL);
-    mip_interface_register_field_callback(&device, &data_handlers[0], MIP_SENSOR_DATA_DESC_SET,
-                                          MIP_DATA_DESC_SENSOR_ACCEL_SCALED, &handleAccel, NULL);
-    mip_interface_register_field_callback(&device, &data_handlers[1], MIP_SENSOR_DATA_DESC_SET,
-                                          MIP_DATA_DESC_SENSOR_GYRO_SCALED, &handleGyro, NULL);
-    mip_interface_register_field_callback(&device, &data_handlers[2], MIP_SENSOR_DATA_DESC_SET,
-                                          MIP_DATA_DESC_SENSOR_MAG_SCALED, &handleMag, NULL);
-    mip_interface_register_extractor(&device, &data_handlers[3], MIP_SENSOR_DATA_DESC_SET,
-                                     MIP_DATA_DESC_SENSOR_ACCEL_SCALED,
-                                     &extract_mip_sensor_scaled_accel_data_from_field, &scaled_accel);
+    mip_interface_register_packet_callback(&device, &packet_handler, MIP_DISPATCH_ANY_DATA_SET, false, &handlePacket, NULL);
+    mip_interface_register_field_callback(&device, &data_handlers[0], MIP_SENSOR_DATA_DESC_SET, MIP_DATA_DESC_SENSOR_ACCEL_SCALED,
+        &handleAccel, NULL);
+    mip_interface_register_field_callback(&device, &data_handlers[1], MIP_SENSOR_DATA_DESC_SET, MIP_DATA_DESC_SENSOR_GYRO_SCALED,
+        &handleGyro, NULL);
+    mip_interface_register_field_callback(&device, &data_handlers[2], MIP_SENSOR_DATA_DESC_SET, MIP_DATA_DESC_SENSOR_MAG_SCALED,
+        &handleMag, NULL);
+    mip_interface_register_extractor(&device, &data_handlers[3], MIP_SENSOR_DATA_DESC_SET, MIP_DATA_DESC_SENSOR_ACCEL_SCALED,
+        &extract_mip_sensor_scaled_accel_data_from_field, &scaled_accel);
 
```

Comment:
In this case, leaving everything on one line makes sense, even if you have to scroll to see it, because the differences between the lines are important. Lining everything up makes it really easy for our eyes to see patterns. For example, what if the last line had a bug, `&data_handlers[2]` instead of `[3]`? If those are all lined up it's really easy to see 0,1,2,2 and realize there's a problem.

Reply:
We also need to consider people who don't have a 4k monitor that can see 500 (over exaggeration) characters on their screen at a time lol. I think we can lineup data_handlers and it'll still be readable without needing to scroll horizontally